nix/overlay.nix

```diff
@@ -97,4 +97,11 @@ in
     buildType = "CI";
     arrow-cpp = final.arrow-cpp-no-simd;
   };
+
+  vast-vm-tests = prev.lib.optionalAttrs prev.stdenv.isLinux (import ./nixos-test.nix
+    {
+      makeTest = import (prev.path + "/nixos/tests/make-test-python.nix");
+      pkgs = final;
+      inherit self;
+    });
```

Comment:
Is there a reason that this is threaded through the overlay? I would think it is better to call the tests directly from `flake.nix`.

Reply:
Agreed.